lib/timescaledb/rails/extensions/active_record/schema_statements.rb

```diff
@@ -40,29 +40,42 @@ def create_hypertable(table_name, time_column_name, **options, &block)
           execute "SELECT create_hypertable('#{table_name}', '#{time_column_name}', #{options_as_sql});"
         end
 
-        # Enables compression and sets compression options.
+        # Enables compression on given hypertable.
         #
-        #   add_hypertable_compression('events', 7.days, segment_by: :created_at, order_by: :name)
+        #   enable_hypertable_compression('events', segment_by: :created_at, order_by: :name)
         #
-        def add_hypertable_compression(table_name, compress_after, segment_by: nil, order_by: nil)
-          compress_after = compress_after.inspect if compress_after.is_a?(ActiveSupport::Duration)
-
+        def enable_hypertable_compression(table_name, segment_by: nil, order_by: nil)
           options = ['timescaledb.compress']
           options << "timescaledb.compress_orderby = '#{order_by}'" unless order_by.nil?
           options << "timescaledb.compress_segmentby = '#{segment_by}'" unless segment_by.nil?
 
           execute "ALTER TABLE #{table_name} SET (#{options.join(', ')});"
+        end
+
+        # Disables compression on given hypertable.
+        #
+        #   disable_hypertable_compression('events')
+        #
+        def disable_hypertable_compression(table_name)
```

Comment:
I think you need to define all enable_hypertable_compression params here, to support rollbacks